sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -48,7 +48,7 @@ private[hive] class SparkExecuteStatementOperation(
     runInBackground: Boolean = true)
     (sqlContext: SQLContext, sessionToActivePool: JMap[SessionHandle, String])
   extends ExecuteStatementOperation(parentSession, statement, confOverlay, runInBackground)
-  with Logging {
+    with Logging {
```

Comment:
nit: revert. https://github.com/databricks/scala-style-guide#spacing-and-indentation `with` shouldn't get extra indentation compared to `extends`

Reply:
my local scala style may have some problem, not fit with that.